Classes/LinphoneCoreSettingsStore.m

```diff
@@ -768,6 +760,8 @@ - (BOOL)synchronize {
         }
 		linphone_core_set_video_preset(lc, [videoPreset UTF8String]);
 		MSVideoSize vsize;
+        vsize.width = 0;
+        vsize.height = 0;
         NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Just the initial initialisation of variable

Xcode 7.3 gives error without this initialization. 
"Variable 'vsize' is used uninitialized whenever 'if' condition is false"